examples/cart/index.js

```diff
@@ -151,48 +167,59 @@ $(function() {
       $('.cart').removeClass('js-active');
     });
   }
-  closeCart();
+
 
   /* Add 'quantity' amount of product 'variant' to cart
   ============================================================ */
-  function addVariantToCart(product, variant, quantity) {
+  function addVariantToCart(variant, quantity) {
     openCart();
 
-    cart.addVariants({ variant: variant, quantity: quantity }).then(function () {
-      var $cartItemContainer = $('.cart-item-container');
-      var totalPrice = 0;
-
-      $cartItemContainer.empty();
-      var lineItemEmptyTemplate = $('#cart-item-template').html();
-      var $cartLineItems = cart.lineItems.map(function (lineItem, index) {
-        var $lineItemTemplate = $(lineItemEmptyTemplate);
-        var itemImage = lineItem.image.src;
-        $lineItemTemplate.find('.cart-item__img').css('background-image', 'url(' + itemImage + ')');
-        $lineItemTemplate.find('.cart-item__title').text(lineItem.title);
-        $lineItemTemplate.find('.cart-item__variant-title').text(lineItem.variant_title);
-        $lineItemTemplate.find('.cart-item__price').text(formatAsMoney(lineItem.line_price));
-        $lineItemTemplate.find('.cart-item__quantity').attr('value', lineItem.quantity);
-        $lineItemTemplate.find('.quantity-decrement').attr('data-variant-id', lineItem.variant_id);
-        $lineItemTemplate.find('.quantity-increment').attr('data-variant-id', lineItem.variant_id);
-
-        if (index === cart.lineItems.length - 1) {
-          $lineItemTemplate.addClass('js-hidden');
-        }
-        return $lineItemTemplate;
-      });
-      $cartItemContainer.append($cartLineItems);
-
-      setTimeout(function () {
-        $cartItemContainer.find('.js-hidden').removeClass('js-hidden');
-      }, 0)
-
-      $('.cart .pricing').text(formatAsMoney(cart.subtotal));
-
+    cart.addVariants({ variant: variant, quantity: quantity }).then(function() {
+      renderCartItems();
     }).catch(function (errors) {
-      console.log("Fail");
-      console.log(errors);
+      console.log('Fail');
+      console.error(errors);
     });
 
+    updateCartTabButton();
+  }
+
+  /* Render the line items currently in the cart */
+  function renderCartItems() {
+    var $cartItemContainer = $('.cart-item-container');
+    var totalPrice = 0;
+
+    $cartItemContainer.empty();
+    var lineItemEmptyTemplate = $('#cart-item-template').html();
+    var $cartLineItems = cart.lineItems.map(function (lineItem, index) {
+      var $lineItemTemplate = $(lineItemEmptyTemplate);
+      var itemImage = lineItem.image.src;
+      $lineItemTemplate.find('.cart-item__img').css('background-image', 'url(' + itemImage + ')');
```

Comment:
Just trying to keep the example as simple and small as possible. The focus of the example is meant to be the interactions with the SDK, so I didn't want to add in anything fancy with templating. I'm open to hearing other options though - whatever makes the example as simple and digestible as possible. 

@richgilbank thoughts?